frontend/.prettierrc

```diff
@@ -0,0 +1,7 @@
+{
+  "trailingComma": "es5",
+  "tabWidth": 2,
+  "semi": true,
+  "singleQuote": false,
+  "printWidth": 100,
+}
```

Comment:
Does this come from some style guide, or is this just your opinion?

Reply:
So we do not have Serokell style guide, but that one comes from tender app